prive/run.py

```diff
@@ -0,0 +1,97 @@
+"""
+Primary command-line access script into the prive library
+"""
+
+import argparse
+from datetime import datetime
+import json
+import os
+
+import numpy as np
+
+from prive.attacks import load_attack
+from prive.datasets import TabularDataset, TabularRecord
+from prive.generators import generator_from_executable
+from prive.threat_models import TargetedAuxiliaryDataMIA
+
+
+def main():
+    parser = argparse.ArgumentParser()
+    parser.add_argument('--genpath', '-G', type=str, help='Path to generator executable')
+    parser.add_argument('--datapath', '-D', type=str,
+                        help='Path to directory containing both csv and json of the test data')
+    parser.add_argument('--auxdatapath', '-AD', type=str, default=None,
```

Comment:
I like the use of both long and short versions of the options.

Would prefer single letters for the short options (eg, -A for --auxdatapath and -C for --runconfig)

Reply:
Will update.